WalletWasabi.Backend/InitConfigStartupTask.cs

```diff
@@ -64,12 +64,18 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
 
 		private static void TaskScheduler_UnobservedTaskException(object? sender, UnobservedTaskExceptionEventArgs e)
 		{
-			Logger.LogWarning(e?.Exception);
+			if (e?.Exception is Exception ex)
+			{
+				Logger.LogWarning(ex);
+			}
```

Comment:
Why not simply remove `?` since `e` is not nullable?
`Logger.LogWarning(e.Exception);`

Reply:
You are right, I fixed it in the original too.